resources/migrations/000_migrations.yaml

```diff
@@ -13073,6 +13073,61 @@ databaseChangeLog:
                     nullable: false
                   defaultValueComputed: current_timestamp
 
+  - changeSet:
+      id: v46.00-028
+      author: tsmacdonald
+      comment: Added 0.46.0 -- Join table connecting cards to dashboards/cards's parameters that need custom filter values from the card
+      changes:
+        - createTable:
+            tableName: values_card
+            remarks: "Join table connecting cards to entities (dashboards, other cards, etc.) that use the values generated by the card for filter values"
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "most recent modification time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "creation time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: card_id
+                  type: int
+                  remarks: "ID of the card generating the values"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: parameterized_object_type
```

Comment:
can we use `parameter_type` and `parameter_id` for these? I think we're dealing with parameters and not objects that have been parameterized.

Reply:
`parameterized_object` is my term for "either a dashboard or a card", which feels true? I specifically can't use `parameter_id` since we _have_ a `parameter_id` that's…exactly what it sounds like (e.g., `(-> dashboard :parameters first :id)` ).

Might help to think of it as being keyed on

```
card_id
dashboard_id
parameter_id
```

except instead of `dashboard_id` we have to use `parameterized_object_{type,id}`

Reply:
ah good distinction. thanks.